tokio-trace/tokio-trace-core/src/callsite.rs

```diff
@@ -37,6 +72,9 @@ pub trait Callsite: Sync {
     /// [dispatcher]: ../dispatcher/struct.Dispatch.html
     fn add_interest(&self, interest: Interest);
 
+    /// Relevant documentation
+    fn set_interest(&self, interest: Interest);
```

Comment:
Since we're making this breaking change, I think we should probably just go ahead and _remove_ the `add_interest` and `clear_interest` methods entirely in favour of `set_interest`.